docs/CONTRIBUTING_DOCS.md

```diff
@@ -84,6 +84,25 @@ where `file.md` is the one to be added. For more information on including files
 To add information in the guide, please add markdown (`.md`) files to the `docs/guide` directory.
 Remember to add new files to the guide's TOC file `docs/source/guide/guide.md`.
 
+### Adding workflow images to the user guide
+
+To insert a workflow to the user's guide of some new technique, a template is available (shown below as a `png`). This template is also available in `svg` format at .. |mitiq_template.svg| image:: ../source/img/general_template.svg
```

Comment:
This is still not linking to the SVG image.

Reply:
```suggestion
To insert a workflow to the user's guide of some new technique, a template is available (shown below as a `png`). This template is also available in `svg` format at [mitiq_template.svg](../source/img/general_template.svg).
```